pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -37,7 +37,7 @@ import (
 // PredicateMetadata interface represents anything that can access a predicate metadata.
 type PredicateMetadata interface {
 	ShallowCopy() PredicateMetadata
-	AddPod(addedPod *v1.Pod, nodeInfo *schedulernodeinfo.NodeInfo) error
+	AddPod(addedPod *v1.Pod, node *v1.Node) error
```

Comment:
Could you elaborate the reason on this change? I don't see the necessity.

Reply:
1. Using more specific arguments makes the meaning of the method clearer and easier to understand ( We actually need `Node` instead of `NodeInfo` ）
2. Keep the code style consistent，as we already used `Node` in `RemovePod` https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/algorithm/predicates/metadata.go#L41

Reply:
SG